src/webapp/pages/import-template/ImportTemplatePage.tsx

```diff
@@ -139,12 +133,23 @@ export default function ImportTemplatePage({ settings }: RouteComponentProps) {
                             const dataSetConfig = {
                                 title: i18n.t("Existing data values"),
                                 message: i18n.t(
-                                    "There are {{totalExisting}} data values in the database for this organisation unit and periods. If you proceed, all those data values will be deleted and only the ones in the spreadsheet will be saved. Are you sure?",
+                                    "There are {{totalExisting}} data values in the database for this organisation unit and periods. All those data values can either be completely deleted or will be kept and only the ones with different values will be updated. What do you want to do?",
                                     { totalExisting }
                                 ),
-                                save: i18n.t("Proceed"),
+                                save: i18n.t("Delete and Import"),
                                 cancel: i18n.t("Cancel"),
                                 info: i18n.t("Import only new data values"),
+                                updateText: i18n.t("Import and Update"),
+                                saveButtonPrimary: false,
+                                saveTooltipText: i18n.t(
+                                    "DELETE ALL EXISTING VALUES AND IMPORT THE ONES IN THE SPREADSHEET"
+                                ),
+                                infoTooltipText: i18n.t(
+                                    "IMPORT ONLY NEW DATA VALUES, WITHOUT UPDATING OR DELETING CURRENT ONES"
+                                ),
+                                updateTooltipText: i18n.t(
+                                    "IMPORT AND UPDATE DATA VALUES WITHOUT DELETING THE REMAINING EXISTING DATA"
```

Comment:
Typically, we use normal case for texts (the css decides how it's shown). But maybe that was a requirement, so ignore this.